README.md

```diff
@@ -588,7 +588,7 @@ Other Style Guides
     }
 
     // good
-    const foo = function bar() {
+    const foo = function foo () {
```

Comment:
this is actually intentional - having the lexically bound name reference be different than the variable containing the function means that things are more greppable, debuggable, and easier to maintain.